extras/rya.benchmark/src/test/java/org/apache/rya/benchmark/query/BenchmarkQueriesReaderIT.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.rya.benchmark.query;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+
+import javax.xml.bind.JAXBException;
+
+import org.apache.rya.benchmark.query.QueryBenchmark.BenchmarkQueriesReader;
+import org.junit.Test;
+import org.xml.sax.SAXException;
+
+import com.google.common.base.Charsets;
+
+/**
+ * Tests the methods of {@link BenchmarkQueriesReader}.
+ */
+public class BenchmarkQueriesReaderIT {
+
+    @Test
+    public void load() throws JAXBException, SAXException {
+        // Unmarshal some XML.
+        final String xml =
+                "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n" +
+                "<BenchmarkQueries>\n" +
+                "    <SPARQL><![CDATA[SELECT ?a WHERE { ?a <http://likes> <urn:icecream> . }]]></SPARQL>" +
```

Comment:
We should have query level granularity for the benchmark (i.e. we should have timing results for each query in the benchmark), so that we can have pretty graphs like this:

![image](https://cloud.githubusercontent.com/assets/4710489/17784519/c838a508-654a-11e6-8d65-e4043177dc6a.png)


Also, it would be nice if the benchmark printed out the name of the query for the results.  So we can have a table like this:

![image](https://cloud.githubusercontent.com/assets/4710489/17784596/14630158-654b-11e6-8dbb-23325619d3d7.png)

(Note: I ripped off these graphics from Roshan's Paper)